src/Symfony/Component/ObjectMapper/Tests/Fixtures/MapTargetToSource/B.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Tests\Fixtures\MapTargetToSource;
+
+use Symfony\Component\ObjectMapper\Attribute\Map;
+
+#[Map(target: self::class, source: A::class)]
```

Comment:
Would it be possible to infer the target in such a case?

```suggestion
#[Map(source: A::class)]
```

Reply:
sure, removed